### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by Strubbl**
i rebased @tcitworld 's PR #19 to master

### CODE REVIEW

**Comment by nicosomb** [comment]
Sorry to ask that ... but is that possible to rebase your PR please?

**Comment by Strubbl** [comment]
I tried to rebase. But when reviewing, please have a closer look on the merge conflict affected files, where i had to solve conflicts manually:

	    en/admin/installation/installation.md
	    en/admin/installation/requirements.md
	    en/admin/maintenance/upgrades/upgrade21-22.md
	    en/developer/docker.md
	    en/user/faq.md

File: en/admin/asynchronous.md [hunk 1]
```diff
@@ -48,13 +46,11 @@ rabbitmq_prefetch_count: 10 # read http://www.rabbitmq.com/consumer-prefetch.htm
...
 
-In internal settings, in the **Import** section, enable RabbitMQ (with
-the value 1).
+In internal settings, in the **Import** section, enable RabbitMQ (with the value 1).
 
 ### Launch RabbitMQ consumer
 
-Depending on which service you want to import from you need to enable
-one (or many if you want to support many) cron job:
+Depending on which service you want to import from you need to enable one (or many if you want to support many) cron job:
```
Comment (Kdecherf) [inline en/admin/asynchronous.md]: Why are you removing 80-char line limits?

Comment (Strubbl) [reply en/admin/asynchronous.md]: it is completely inconsistent all over this repo

**Comment by j0k3r** [comment]
We forgot about this PR @Strubbl 😞
Any chance you give a try to rebase it?

**Pull request closed without merging**